worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -227,6 +234,26 @@ private int processHandles(List<Future<EobSearchResult>> futureHandles, Progress
         return numberOfEobs;
     }
 
+    void addMbiIdToEobs(List<ExplanationOfBenefit> eobs, Map<String, PatientDTO> patients) {
+        if (eobs == null || eobs.isEmpty()) {
+            return;
+        }
+        // Get first EOB Bene ID
+        ExplanationOfBenefit eob = eobs.get(0);
+        String mbi = null;
+        String benId = getPatientIdFromEOB(eob);
+        if (benId != null) {
+            mbi = patients.get(benId).getMbiId();
+        }
+        Extension extension = createExtension(mbi);
```

Comment:
If benId is null then we should be skipping this correct or is benId never null?

Reply:
beneId should never be null, but if benId is null then there is no way to find the mbi id anyway why try and then have a null pointer exception.